package.json

```diff
@@ -236,14 +256,10 @@
         "permissionName": "ui-circulation.settings.lost-item-fees-policies",
         "displayName": "Settings (Circ): Can create, edit and remove lost item fee policies",
         "subPermissions": [
+          "ui-circulation.settings.view-lost-item-fees-policies",
           "lost-item-fees-policies.item.post",
           "lost-item-fees-policies.item.put",
-          "lost-item-fees-policies.item.delete",
```

Comment:
Could you please confirm that we need delete "lost-item-fees-policies.item.delete" permissions here?

Reply:
I did not make any substantive changes to the "Can create, edit and remove..." policy, only restructured it so the "view" values are now pulled in indirectly via the "view-only" policy. It appears as changed lines only because `delete` is now the last value and therefore has no trailing comma.

Note [line 262](https://github.com/folio-org/ui-circulation/blob/0c5138227812f0747cef4c0ef1932b79b1a8f4da/package.json#L262)